app-editors/vscode/vscode-1.77.1.ebuild

```diff
@@ -122,7 +122,5 @@ src_install() {
 
 pkg_postinst() {
 	xdg_pkg_postinst
-	elog "You may want to install some additional utils, check in:"
```

Comment:
You're sure you want to remove it entirely, rather than only showing it on first emerge?

Reply:
Its my personal opinion, but I think most people know how vscode works. I don't think it should be shown in the first emerge as well. It just wastes time.